audobject/core/object.py

```diff
@@ -26,6 +26,12 @@ class Object:
           bar: hello object!
 
     """
+    def __init__(
+            self,
+            **kwargs,
+    ):
+        self.__dict__[define.KEYWORD_ARGUMENTS] = list(kwargs)
```

Comment:
Do we have to store it inside `__dict__` or could we also use a hidden dictionary instead?
The problem with `__dict__` is that it requires the workarounds in the implementation of `audobject.Dictionary`

Reply:
Correct me if I'm wrong but as far as I know `__dict__` is the only place where we can store attributes.

But what we could do is group the special attributes into a dictionary under `__dict__['_object_'][...]`.

Reply:
But I suggest we do that in a separate PR.

Reply:
Yeah, I looked into `object.py` and there we use `__dict__` all over the place.

`__dict__['_object_']` would also not help with the code inside `audobject.Dictionary`, so I would propose we stay with the current solution.